bigtable/tests/unit/test_table.py

```diff
@@ -569,6 +616,11 @@ def _SampleRowKeysRequestPB(*args, **kw):
 
     return messages_v2_pb2.SampleRowKeysRequest(*args, **kw)
 
+def _MutateRowsRequestPB(*args, **kw):
```

Comment:
- Please us PEP8 function names (`_mutate_rows_request_pb`) (I realize there is already a violation of that in this file but let's not pile on)
- Two empty lines between top-level objects (so you need an extra one before this function and after it)